buildbot/master/files/config/master.cfg

```diff
@@ -8,10 +8,11 @@ from passwords import SLAVE_PASSWORD, CHANGE_PASSWORD
 from passwords import HOMU_BUILDBOT_SECRET, GITHUB_STATUS_TOKEN
 
 
-LINUX_SLAVES = ["servo-linux1", "servo-linux2", "servo-linux3"]
+LINUX_SLAVES = ["servo-linux1", "servo-linux2", "servo-linux3", "servo-linux4",
```

Comment:
IMO this is a bit ugly, something similar to the `builderNames` arrays looks nicer, although I don't mind multiple names per line.